components/kyma-environment-broker/internal/process/input/builder_test.go

```diff
@@ -215,6 +216,70 @@ func TestInputBuilderFactory_ForPlan(t *testing.T) {
 		assert.Equal(t, gqlschema.KymaProfileEvaluation, *result.upgradeRuntimeInput.KymaConfig.Profile)
 	})
 
+	t.Run("should build CreateUpgradeShootInput with proper plan", func(t *testing.T) {
```

Comment:
let's name it: should build CreateUpgradeShootInput with proper autoscaler parameters

Reply:
please, create a test similar to that, but provisioning parameters should contains autoscaler params. such test will show us that the user's values are used instead of defaults